### PR TITLE
bulk selection updates

### LINKED ISSUE #1816: Update images on Bulk Selection page
Update to RH font
Update to new PF blue
Update to status icons in table

### PR DESCRIPTION
**Pull request by gdoyle1**
ongoing updates for #1816 

closes #1816

### CODE REVIEW

**Comment by gdoyle1** [comment]
@mmenestr this is finally ready to go!

File: packages/v4/src/content/design-guidelines/usage-and-behavior/bulk-selection/bulk-selection.md [hunk 1]
```diff
@@ -5,20 +5,18 @@ hideTOC: true
 showTitle: true
 ---
 
-
-
 Use a bulk selection pattern when you want to select or deselect multiple items in a content view (list, table, or card grid). The bulk selector uses a [Split button](/documentation/react/components/dropdown#split-button) component to control selection from the [Toolbar](/documentation/react/beta/datatoolbar). Besides controlling selection, the bulk selector reflects the selection status of the related component (partially selected, all items selected, or no items selected).
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/usage-and-behavior/bulk-selection/bulk-selection.md:8]: Toolbar link broken

File: packages/v4/src/content/design-guidelines/usage-and-behavior/bulk-selection/bulk-selection.md [hunk 2]
```diff
@@ -5,20 +5,18 @@ hideTOC: true
...
 
-![bulk selector](./img/bulk-selector.png)
+<img src="./img/bulk-selector.png" alt="Bulk selector" width="1449"/>
 
 **1. Bulk selector:** The bulk selector uses a split button to combine a selection checkbox with a dropdown menu.
 
 **2. Checkbox:** The state of the checkbox reflects the current selection state of the list, table, or card grid.
 
-![blulk selector checkbox](./img/bulk-select-checkbox.png)
+<img src="./img/bulk-select-checkbox" alt="Bulk Selector Checkbox States" width="518"/>
```
Comment (mmenestr) [inline packages/v4/src/content/design-guidelines/usage-and-behavior/bulk-selection/bulk-selection.md:19]: image issue

**Comment by gdoyle1** [comment]
@mmenestr made the fixes!

**Pull request merged**